aten/src/ATen/native/mps/operations/LinearAlgebra.mm

```diff
@@ -125,17 +125,11 @@ void prepare_matrices_for_broadcasting(
 
   MPSStream* stream = getCurrentMPSStream();
 
-  bool transpose_mat1            = false;
-  bool transpose_mat2            = false;
-
-  prepare_matrices_for_broadcasting(NULL, self, other, NULL, NULL, transpose_mat1, transpose_mat2);
```

Comment:
I am not following this? we don't need the prepare_matrices_for_broadcasting? How are we handling transposes.